tests/test_bot.py

```
@@ -93,6 +93,25 @@ async def cmd(ctx):
     assert kwargs["guild"].id == 12345
 
 
+def test_slash_nsfw_flag_is_forwarded(bot):
+    @bot.slash(description="Secret", nsfw=True)
+    async def secret(ctx):
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `secret`

Add return type annotation: `None`

(ANN202)